swagger2.0/oic.wk.res.swagger.json

```diff
@@ -251,6 +251,17 @@
             "items": {
               "$ref": "#/definitions/oic.oic-link"
             }
+          },
+          "sduuid": {
+            "description": "A UUID that identifies the Security Domain.",
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
```

Comment:
Rather than specifying the "pattern" here, shouldn't this reference http://openconnectivityfoundation.github.io/core/schemas/oic.types-schema.json#/definitions/uuid